bin/alluxio-workers.sh

```diff
@@ -30,7 +30,9 @@ ALLUXIO_LIBEXEC_DIR=${ALLUXIO_LIBEXEC_DIR:-${DEFAULT_LIBEXEC_DIR}}
 . ${ALLUXIO_LIBEXEC_DIR}/alluxio-config.sh
 
 HOSTLIST=$(cat ${ALLUXIO_CONF_DIR}/workers | sed  "s/#.*$//;/^$/d")
-ALLUXIO_TASK_LOG="$(echo ${BIN} | sed 's/bin$//g')"logs/task.log
+ALLUXIO_TASK_LOG_DIR="${BIN//bin/}logs"
```

Comment:
`ALLUXIO_TASK_LOG_DIR="${BIN}/../logs"` for consistency with how this is done elsewhere in this file